src/test/test.cpp

```diff
@@ -168,14 +170,17 @@ test(int argc, char* const* argv, el::Level ll,
     using namespace Catch;
     Session session{};
 
-    session.cli(
-        session.cli() |
-        clara::Opt(gTestAllVersions)["--all-versions"]("Test all versions") |
-        clara::Opt(gVersionsToTest,
-                   "version")["--version"]("Test specific version(s)"));
+    auto cli = session.cli();
+    cli |= clara::Opt(gTestAllVersions)["--all-versions"]("Test all versions");
+    cli |= clara::Opt(gVersionsToTest,
+                      "version")["--version"]("Test specific version(s)");
+    cli |= clara::Opt([](int x) { gBaseInstance = x; },
```

Comment:
no need for lambda, it should be
`clara::Opt(gBaseInstance, "base-instance")["--base-instance"] ...`